functional-tests/devscripts/run_tests.sh

```diff
@@ -97,7 +97,7 @@ if [[ "$PR_CHECK_BUILD" == "true" ]]; then
   #check version in rh-che pom.xml to use correct version of tests.
   getMavenVersion # Checking the maven version for debugging https://github.com/redhat-developer/rh-che/issues/1716
   version=$(getVersionFromPom)
-  if [[ -z "${version}" ]]; then
+  if [[ -z "${version}" || $version = "" ]]; then
```

Comment:
Is this really a necessary change?

Reply:
`if [ -z "${version}"]; then` should do the trick

Reply:
Tests are failing if the version is empty. It's probably not caught by the `-z` switch.

Reply:
I see, I'll remove it.

Reply:
you need to have a single `[` in the if statement

Reply:
Done.